index.html

```diff
@@ -0,0 +1,203 @@
+          <h1 class="topic1">UGANDA  ANNUAL CULTURAL FORUM</h1>
+        </div>
+        <div class="text">
+          <p class="text1">An Annual discourse, digest and celebration of the tourism and culture sectors of Uganda, the Pearl of Africa...A deep dive into our unique culture and beautiful nation!!! </p>
+        </div>
+        <div class="date">
+          <h4 class="date1">20/10/2020(mon)-23(WED)</h4>
+        </div>
+        <p class="location">@Uganda National House</p>
+      </div>
+    </div>
+  </section>
+
+  <!-- ....//programme//... -->
+  <section id="programme">
+    <div class="prog">
+    <h2 class="prog1">Main Programme</h2>
+    <hr style="height:2px; width: 40px;border-width:0;color:#ff4500;background-color:#ff4500;">
+    </div>
+ <div class="progos">
+  <div class="lectures progo">
+    <img src="images/prog1.png" alt="decorative" class="img1">
+    <h3 class="sub1">Lectures</h3>
+    <div class="para1">
+    <p class="texts">Listen to lectures from our speakers from all over Uganda</p>
+    </div>
+  </div>
+
+  <div class="presentation progo">
+    <img src="images/prog2.png" alt="decorative" class="img2">
+    <h3 class="sub2">Presents</h3>
+    <div class="para2">
+    <p class="texts">Attend fun photo presentations from different visitors.</p>
+    </div>
+  </div>
+
+  <div class="meet progo">
+    <img src="images/prog3.png" alt="decorative" class="img3">
+    <h3 class="sub3">Meet</h3>
+    <div class="para3">
+    <p class="texts">Meet and greet our key speakers and other guests.</p>
+    </div>
+  </div>
+
+  <div class="connect progo">
+    <img src="images/prog4.png" alt="decorative" class="img4">
+    <h3 class="sub4">Connect</h3>
+    <div class="para4">
+    <p class="texts">Connect with Ugandans from all around the country...</p>
+    </div>
+  </div>
+  </div>
+  <button type="button" class="btn btn-danger btn-lg">Join Us Now</button>
+  <h4 class="see">SEE THE WHOLE PROGRAMME</h4>
+
+</section>
+<script src="scripts.js"></script>
+
+
+<!-- ...///partners...// -->
+<section id="partners">
+<div class="partner-text">
+  <h2>Partners</h2>
+  <hr style="height:2px; width: 40px;border-width:0;color:#ff4500;background-color:#ff4500;">
+</div>
+<div class="partner-imgs">
+  <div class="partner-img">
+    <img src="images/logo1.svg" alt="">
+  </div>
+  <div class="partner-img">
+    <img src="images/logo2.svg" alt="">
+  </div>
+  <div class="partner-img">
+    <img src="images/logo3.svg" alt="">
+  </div>
+  <div class="partner-img">
+      <img src="images/logo4.svg" alt="">
+  </div>
+</div>
+</section>
+
+<!-- ....//copyright.../ -->
+<footer class="copy">
+  <img src="images/red.png" alt="logo" class="footer-logo">
+  <div class="c-f">
+ <p class="copy-footer">@danielmatama copyright</p>
+ <p class="copy-footer2">Matama reserves the right to all copies for this website</p>
+   </div>
+</footer>
+
+<script src="scripts.js"></script>
+ </body>
+</html>
+
+
+
+<!-- <script src="scripts.js"></script> -->
```

Comment:
[OPTIONAL] Please consider deleting unused code for it makes you code base look untidy